AuthenticationSimpleSample/README.md

```diff
@@ -0,0 +1,13 @@
+---
+name: .NET MAUI - Authentication Azure AD B2C
+description: "This sample demonstrates how to implement authentication using Azure AD B2C (using external platforms)"
+page_type: sample
+languages:
+- csharp
+- xaml
+products:
+- dotnet-maui
+- dotnet-core
+urlFragment: Authentication
+---
+
```

Comment:
As with the MIAUI sample, let's take some screenshots and reference them in the README here as well